### PR TITLE
Added support for JXS file format

### PR DESCRIPTION
**Pull request by V-R-Dighe**

### CODE REVIEW

**Comment by V-R-Dighe** [comment]
Hey @gabriel-vasile, I am requesting permission to run the workflows. Also, please look into my PR. Thanks!

File: internal/magic/magic.go [hunk 1]
```diff
@@ -177,7 +177,9 @@ func newXMLSig(localName, xmlns string) xmlSig {
 // and, optionally, followed by the arguments for the interpreter.
 //
 // Ex:
-//  #! /usr/bin/env php
+//
+//	#! /usr/bin/env php
+//
```
Comment (gabriel-vasile) [inline internal/magic/magic.go:182]: Please remove this change. It has nothing to do with the scope of the PR.

**Review by gabriel-vasile — COMMENTED** [review]
Hi, PR looks good, except for 2 problems.

1. there is already an [older open PR for JXR](https://github.com/gabriel-vasile/mimetype/pull/312)
2. the PR does not have testfiles 

If you are keen to get this merged ASAP, then make this PR only about JXS format and add a testfile for it. I recommend you convert [the JPG testfile](https://github.com/gabriel-vasile/mimetype/blob/master/testdata/jpg.jpg) to JXS but any other image works as long as it is released under a permissive license and it's under 300kb in size.

File: internal/magic/magic.go (hunk 1 above)
Comment (V-R-Dighe) [reply internal/magic/magic.go]: Hello @gabriel-vasile,
I have made the suggested changes but I am not able to source a JXS image file also I didn't find any converter to convert JPG test file to JXS. Can you help me in getting JXS test image?

**Comment by gabriel-vasile** [comment]
I did some digging about this format. JXS is still in early stages. They decided on the magic numbers and the MIME type of the format, but they didn't yet release an actual implementation of JXS encoder. They will do this sometime in 2022 in Part 5 of the release. 

_Part 5 (ISO/IEC 21122-5) represents a reference software implementation (written in ISO C11) of the JPEG XS Part 1 decoder, conforming to the Part 2 profiles, levels and sublevels, as well as an exemplary encoder implementation._
Check here: https://en.wikipedia.org/wiki/JPEG_XS#Standards

**Review by gabriel-vasile — APPROVED** [review]

**Review by gabriel-vasile — APPROVED** [review]

**Review by gabriel-vasile — APPROVED** [review]

**Comment by V-R-Dighe** [comment]
> I opened #322 as a reminder and I will proceed to merge this as it is.

Thank you @gabriel-vasile.
Surely, I will add the test file once the JXS implementation is released.

**Pull request merged**